TLM/TLM/UI/TrafficManagerTool.cs

```diff
@@ -378,6 +383,8 @@ public class TrafficManagerTool
         /// </summary>
         void DefaultRenderOverlay(RenderManager.CameraInfo cameraInfo)
         {
+            if (!LoadingExtension.PlayMode)
```

Comment:
wrap with `{ }`
i didn't comment this in Harmony files, Harmony is like next level of complexity which i am happy to not see 😅